### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by FelixLetheren**
New changes to push to master

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,38 +24,44 @@ <h1>Portfolio</h1>
     </div>
     <div class="port-container">
         <div class="port-item">
-            <img src="https://picsum.photos/400/300">
-            <h3>Title</h3>
-            <h4>Description kjasndkjjansdn kjansd asjkdbakjsdn jaksbdkja asjkbaskjd kjansdkjn ajksdnkasd</h4>
+            <a href="https://github.com/FelixLetheren/mayden-academy-logo"><img class="port-img"
+                                                                                src="images/mayden-logo.png"></a>
+            <a href="https://github.com/FelixLetheren/mayden-academy-logo"><h3>Mayden Academy logo</h3></a>
+            <h4>A small project to recreate the Myaden Academy logo using just css and html</h4>
```
Comment (sjr-mayden) [inline index.html]: CSS and HTML should be capitalised

Comment (sjr-mayden) [inline index.html]: Myaden Academy

File: stylesheets/stylesheet.css [hunk 2]
```diff
@@ -12,6 +12,7 @@ header {
     background-color: #003049;
     position: sticky;
     top: 0;
+    z-index: 1;
```
Comment (sjr-mayden) [inline stylesheets/stylesheet.css:15]: Discussion: no longer needed

File: stylesheets/stylesheet.css [hunk 3]
```diff
@@ -136,6 +138,27 @@ a {
...
+.port-item h3 {
+    margin-top: 5px;
+}
+
+.port-img:hover {
+    webkit-filter: blur(4px);
+    filter: blur(4px);
+}
+
+.img-cover {
```
Comment (sjr-mayden) [inline stylesheets/stylesheet.css]: Discussion: To be removed

File: stylesheets/stylesheet.css [hunk 4]
```diff
@@ -158,6 +181,17 @@ a {
...
 
+.skill-container {
+    display: flex;
+    justify-content: space-around;
+}
+
+.skill-img {
+    height: 80px;
+    width: 80px;
+
```
Comment (sjr-mayden) [inline stylesheets/stylesheet.css]: Extra newline

File: stylesheets/stylesheet.css [hunk 5]
```diff
@@ -200,8 +236,26 @@ a {
...
+.social-item img {
+    transition: all .2s ease-in-out;
+}
+
+.social-item img:hover {
+    transform: scale(1.1);
+}
+
+
```
Comment (sjr-mayden) [inline stylesheets/stylesheet.css]: Extra newline

File: index.html [hunk 6]
```diff
@@ -24,38 +24,44 @@ <h1>Portfolio</h1>
...
         </div>
         <div class="port-item">
-            <img src="https://picsum.photos/400/300">
-            <h3>Title</h3>
-            <h4>Description kjasndkjjansdn kjansd asjkdbakjsdn jaksbdkja asjkbaskjd kjansdkjn ajksdnkasd</h4>
+            <a href="#"><img class="port-img" src="images/placeholder.png"></a>
```
Comment (sjr-mayden) [inline index.html]: replace "port-" with "portfolio-"

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (sjr-mayden) [reply index.html]: Done

Comment (sjr-mayden) [reply index.html]: Done

File: stylesheets/stylesheet.css (hunk 3 above)
Comment (sjr-mayden) [reply stylesheets/stylesheet.css]: Fine

File: stylesheets/stylesheet.css (hunk 4 above)
Comment (sjr-mayden) [reply stylesheets/stylesheet.css]: Done

File: stylesheets/stylesheet.css (hunk 5 above)
Comment (sjr-mayden) [reply stylesheets/stylesheet.css]: Done

**Comment by sjr-mayden** [comment]
Not approved, shonky pointer

File: contact.html [hunk 7]
```diff
@@ -13,7 +13,9 @@ <h2 class="logo">FELIX<br>LETHEREN</h2>
     <div class="nav-container">
         <div class="nav-bar">
             <a href="index.html" class="nav-items">Portfolio</a>
-            <a href="about.html" class="nav-items">About Me</a>
+            <a href="about.html" class="nav-items">About Me
+                <div class="pointer-arrow" id="contact-pointer"></div>
```
Comment (sjr-mayden) [inline contact.html:18]: Arrow links to about me section regardless of section that you're on

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: contact.html (hunk 7 above)
Comment (FelixLetheren) [reply contact.html:18]: Resolved!

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**